pypeit/par/pypeitpar.py

```diff
@@ -645,7 +645,7 @@ def __init__(self, method=None, pixelflat_file=None, spec_samp_fine=None,
 
         defaults['slit_illum_relative'] = False
         dtypes['slit_illum_relative'] = bool
-        descr['slit_illum_relative'] = 'Generate an image of the relative spectral illumination' \
+        descr['slit_illum_relative'] = 'Generate an image of the relative spectral illumination ' \
```

Comment:
Can you add to these docs that this functionality is currently only used for IFUs?

Reply:
I have added this, but I think @profxj and @marlageha may be using this to account for vignetting of DEIMOS (but I could be mistaken).